packages/react-components/react-shared-contexts/src/AnnounceContext/AnnounceContext.ts

```diff
@@ -0,0 +1,26 @@
+import * as React from 'react';
+
+export type AnnounceOptions = {
+  alert?: boolean;
+  priority?: number;
+  batchId?: string;
```

Comment:
I think we should add `polite?: boolean` here, since we'll definitely need it in the future.

Also, just for stylistic reasons, it'd be nice to alphabetize these